src/bindings/interface/importer.i

```diff
@@ -6,6 +6,8 @@
 %import "logger.i"
 %import "types.i"
 
+%include <std_string.i>
+
```

Comment:
This should be moved after `#define LIBCELLML_EXPORT` as we do in our other interface files.

Reply:
8f04142a